DataDog/storedog
# Throw error in ads service via code flag

## Description
https://datadoghq.atlassian.net/browse/WEB-3213

This PR updates the `/ads` route to optionally return an error. This is configured via the feature flag system `code-stash`, managed by the `featureFlags.config.json` file.

Note
- console logs will be removed prior to merge
- feature flag will be turned off prior to merge

## How to test
1. Pull down branch
2. Rebuild frontend and ads service
`docker-compose build frontend ads`
3. Start the app via docker-compose up 
4. When all services are healthy, visit http://localhost
5. Continue refreshing the homepage to generate errors
6. In the logs for the `ads` service, you should see 500s
7. You should also see 500s in the dev console of the browser

**Expected APM traces view**
![Screen Shot 2023-04-05 at 1 22 02 PM](https://user-images.githubusercontent.com/9119168/230156834-7f096236-733a-4362-9c14-967cc88e78c3.png)


**Expected Error tracking view**
![Screen Shot 2023-04-05 at 1 21 52 PM](https://user-images.githubusercontent.com/9119168/230156855-cca33eb2-39f8-4da0-aec0-e66b74ec36f7.png)
